<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelFeaturesDialog</class>
 <widget class="QDialog" name="ModelFeaturesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Model Features</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="4">
    <widget class="QListView" name="listViewHide"/>
   </item>
   <item row="0" column="2" rowspan="4">
    <widget class="QListView" name="listViewShow"/>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushButtonTop">
     <property name="text">
      <string>Top</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonShow">
     <property name="text">
      <string>Show --&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButtonUp">
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButtonHide">
     <property name="text">
      <string>&lt;-- Hide</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pushButtonDown">
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushButtonBottom">
     <property name="text">
      <string>Bottom</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButtonDone">
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="pushButtonColor">
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listViewHide</tabstop>
  <tabstop>pushButtonShow</tabstop>
  <tabstop>pushButtonHide</tabstop>
  <tabstop>listViewShow</tabstop>
  <tabstop>pushButtonTop</tabstop>
  <tabstop>pushButtonUp</tabstop>
  <tabstop>pushButtonDown</tabstop>
  <tabstop>pushButtonBottom</tabstop>
  <tabstop>pushButtonColor</tabstop>
  <tabstop>pushButtonDone</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
